packages/contracts/tasks/take-dump.ts

```diff
@@ -69,6 +69,13 @@ task('take-dump').setAction(async (args, hre) => {
       _name: 'Ether',
       _symbol: 'ETH',
     },
+    BVM_BIT: {
+      l2Bridge: predeploys.L2StandardBridge,
+      // l1Token: hre.deployConfig.l1BitAddress,
+      l1Token: '0x1A4b46696b2bB4794Eb3D4c26f1c55F9170fa4C5',
```

Comment:
On different enviroments, the `l1Token` will be different.?  Maybe we use `l1Token: hre.deployConfig.l1BitAddress` to insteand of a constant address?

Reply:
it bind 0x1A4b46696b2bB4794Eb3D4c26f1c55F9170fa4C5 on deposit side, all layer2 bit will bind to this address